### ISSUE TITLE
`<object>` inner text should count as alt text

### ISSUE THREAD

**Issue by adamraine** [dequelabs/axe-core#3318]
From https://github.com/GoogleChrome/lighthouse/issues/12528

Product: axe-core

Expectation: `<object>Alt text</object>` Qualifies as alt text

Actual: `<object>Alt text</object>` is flagged by `object-alt`

Motivation: Inner text should count as alt text according to https://dequeuniversity.com/rules/axe/4.0/object-alt.

```
axe-core version: 4.3.5
```

**Comment by WilcoFiers** [comment]
@adamraine Thanks for reporting this. Unfortunately the Deque University page is wrong on this one. This technique was written for WCAG, but never actually worked. You'll need to use one of the other options, like using an aria-labelledby attribute.

**Comment by adamraine** [comment]
Thanks for the clarification @WilcoFiers

**Comment by yecril71pl** [comment]
> This technique was written for WCAG, but never actually worked. 

What do you mean by *never worked*?

**Comment by WilcoFiers** [comment]
WCAG technique [H53 Using the body of the `object` element](https://www.w3.org/WAI/WCAG21/Techniques/html/H53.html) is not supported by assistive technologies. Later when the accessible name computation became standardised, this wasn't included. So it's unlikely this is ever going to work. There's an open issue for it in the WCAG repo: w3c/wcag#1138

**Comment by kantoche** [comment]
Could the description of `object-alt` rule be fixed accordingly at https://dequeuniversity.com/rules/axe/4.4/object-alt ? 
Description still mentions :"Add alternative text to all embedded <object> elements using **either inner text**, title attributes, aria-label or aria-labelledby" and the examples have not been updated.
